js/adslot.jsx

```diff
@@ -1,75 +1,80 @@
-
 import React from 'react';
+import PropTypes from 'prop-types';
 import { DFPManager } from './manager';
 
+let dynamicAdCount = 0;
 
-export const AdSlot = React.createClass({
-  displayName: 'AdSlot',
+export class AdSlot extends React.Component {
 
-  propTypes: {
-    dfpNetworkId: React.PropTypes.string,
-    adUnit: React.PropTypes.string,
-    sizes: React.PropTypes.arrayOf(
-      React.PropTypes.oneOfType([
-        React.PropTypes.arrayOf(React.PropTypes.number),
-        React.PropTypes.string,
-      ])
+  static propTypes = {
```

Comment:
I always look at code and am all, "What asshole wrote this?" Then I look at the commits and I realize it's me from six months ago.